docs/sources/setup-grafana/configure-security/export-logs.md

```diff
@@ -32,7 +32,11 @@ Usage insights logs are JSON objects that represent certain user activities, suc
 
 ### Scope
 
-A log is created every time a user opens a dashboard or when a query is sent to a data source in the dashboard view. A query that is performed via Explore does not generate a log.
+A log is created every time:
+
+- A user opens a dashboard.
+- A query is sent to a data source in the dashboard view.
+- A query is performed via Explore.
```

Comment:
Do we need to include the extra context from **[here](https://github.com/grafana/grafana/pull/59931#issue-1481327213)**:

> If an Explore query has an error, then that error is ignored but an event is still recorded. Should Grafana instead not send an event at all to Usage Insights if the Explore query returned an error?

?